### PR TITLE
Local Discovery tab and New nav icons

### PR DESCRIPTION
**Pull request by l2succes**
Depends on https://github.com/artsy/emission/pull/1264

- Updates Navigation Icons
- Adds Local Discovery Tab behind a lab option

Still need to update Podfile.lock with latest emission version once published

### CODE REVIEW

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5275,8 +5283,8 @@
 				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/Unica77LL-Regular.otf",
 				"${PODS_ROOT}/Artsy+UILabels/Pod/Assets/Chevron_Black.png",
 				"${PODS_ROOT}/Artsy+UILabels/Pod/Assets/Chevron_Black@2x.png",
-				"${PODS_ROOT}/Emission/Pod/Assets/Emission.js",
-				"${PODS_ROOT}/Emission/Pod/Assets/assets",
+				"${PODS_ROOT}/../../emission/Pod/Assets/Emission.js",
```
Comment (l2succes) [inline Artsy.xcodeproj/project.pbxproj]: Will be updated once pointing to non-local emission version

File: Podfile [hunk 2]
```diff
@@ -93,6 +93,7 @@ target 'Artsy' do
   # my fork let's the scripts folder exist inside the repo so that it can be executed in the post_install below
   pod 'react-native-mapbox-gl', git: 'https://github.com/orta/react-native-mapbox-gl.git', branch: '6_1_3_podspec_fix'
   pod 'SentryReactNative', git: 'https://github.com/getsentry/react-native-sentry.git', tag: 'v0.30.3'
+  pod 'Pulley', path: '../Pulley'
```
Comment (l2succes) [inline Podfile]: will remove

File: Podfile [hunk 3]
```diff
@@ -80,7 +80,7 @@ target 'Artsy' do
   pod 'Artsy+UILabels'
   pod 'Extraction'
 
-  pod 'Emission', '~> 1.7.0'
+  pod 'Emission', path: '../emission'
```
Comment (l2succes) [inline Podfile]: Should point to latest emission once ready

File: Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m [hunk 4]
```diff
@@ -286,6 +287,7 @@ - (void)registerWithSwitchBoard:(ARSwitchBoard *)switchboard
     NSDictionary *menuToPaths = @{
         @(ARTopTabControllerIndexHome) : @"/",
         @(ARTopTabControllerIndexMessaging) : @"/inbox",
+        @(ARTopTabControllerIndexLocalDiscovery) : @"/local-discovery",
```
Comment (l2succes) [inline Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m]: Not sure if this is needed

**Comment by orta** [comment]
Yep, this looks good to me- nice work

**Comment by orta** [comment]
OK, cool 👍

File: Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m (hunk 4 above)
Comment (orta) [reply Artsy/View_Controllers/App_Navigation/ARTopMenuViewController.m]: It is, yep

File: Artsy/App/ARAppDelegate+Emission.m [hunk 5]
```diff
@@ -115,6 +115,7 @@ - (void)setupSharedEmissionWithPackagerURL:(NSURL *)packagerURL;
                                                                                 sentryDSN:sentryDSN
                                                                      stripePublishableKey:stripePublishableKey
                                                                          googleMapsAPIKey:[keys googleMapsAPIKey]
+                                                                       mapBoxAPIClientKey:[keys mapBoxAPIClientKey]
```
Comment (orta) [inline Artsy/App/ARAppDelegate+Emission.m]: this needs to be added the CocoaPods keys, doing this now

**Pull request merged**